### ISSUE TITLE
[BUG] Wizard uses corrupt version of setuptools atm

### ISSUE THREAD

**Issue by eorituz** [raiden-network/raiden-wizard#95]
## Issue
The latest version of setuptools causes the built executable binary to crash.
```
(wizard) oliver@bbpc:~/Arbeit/github/raiden-wizard$ mv /home/oliver/Arbeit/github/raiden-wizard/dist/raiden_wizard .
(wizard) oliver@bbpc:~/Arbeit/github/raiden-wizard$ ./raiden_wizard 
[14878] PyInstaller Bootloader 3.x
[14878] LOADER: executable is /home/oliver/Arbeit/github/raiden-wizard/raiden_wizard
[14878] LOADER: homepath is /home/oliver/Arbeit/github/raiden-wizard
[14878] LOADER: _MEIPASS2 is NULL
[14878] LOADER: archivename is /home/oliver/Arbeit/github/raiden-wizard/raiden_wizard
[14878] LOADER: Extracting binaries
[14878] LOADER: Executing self as child
[14878] LOADER: set _MEIPASS2 to /tmp/_MEIJUGdyZ
[14878] LOADER: LD_LIBRARY_PATH=/tmp/_MEIJUGdyZ
[14878] LOADER: Registering signal handlers
[14879] PyInstaller Bootloader 3.x
[14879] LOADER: executable is /home/oliver/Arbeit/github/raiden-wizard/raiden_wizard
[14879] LOADER: homepath is /home/oliver/Arbeit/github/raiden-wizard
[14879] LOADER: _MEIPASS2 is /tmp/_MEIJUGdyZ
[14879] LOADER: archivename is /home/oliver/Arbeit/github/raiden-wizard/raiden_wizard
[14879] LOADER: Already in the child - running user's code.
[14879] LOADER: Python library: /tmp/_MEIJUGdyZ/libpython3.7m.so.1.0
[14879] LOADER: Loaded functions from Python library.
[14879] LOADER: Manipulating environment (sys.path, sys.prefix)
[14879] LOADER: sys.prefix is /tmp/_MEIJUGdyZ
[14879] LOADER: Pre-init sys.path is /tmp/_MEIJUGdyZ/base_library.zip:/tmp/_MEIJUGdyZ
[14879] LOADER: Setting runtime options
[14879] LOADER: Initializing python
[14879] LOADER: Overriding Python's sys.path
[14879] LOADER: Post-init sys.path is /tmp/_MEIJUGdyZ/base_library.zip:/tmp/_MEIJUGdyZ
[14879] LOADER: Setting sys.argv
[14879] LOADER: setting sys._MEIPASS
[14879] LOADER: importing modules from CArchive
[14879] LOADER: extracted struct
[14879] LOADER: callfunction returned...
[14879] LOADER: extracted pyimod01_os_path
[14879] LOADER: callfunction returned...
[14879] LOADER: extracted pyimod02_archive
[14879] LOADER: callfunction returned...
[14879] LOADER: extracted pyimod03_importers
[14879] LOADER: callfunction returned...
[14879] LOADER: Installing PYZ archive with Python modules.
[14879] LOADER: PYZ archive: PYZ-00.pyz
[14879] LOADER: Running pyiboot01_bootstrap.py
[14879] LOADER: Running runtime_raiden_contracts.py
Traceback (most recent call last):
  File "tools/pyinstaller/hooks/runtime_raiden_contracts.py", line 3, in <module>
    from raiden_contracts import contract_manager
  File "/home/oliver/.virtualenvs/wizard/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
    exec(bytecode, module.__dict__)
  File "site-packages/raiden_contracts/contract_manager.py", line 11, in <module>
  File "/home/oliver/.virtualenvs/wizard/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
    exec(bytecode, module.__dict__)
  File "site-packages/raiden_contracts/constants.py", line 6, in <module>
  File "/home/oliver/.virtualenvs/wizard/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
    exec(bytecode, module.__dict__)
  File "site-packages/eth_utils/__init__.py", line 1, in <module>
  File "/home/oliver/.virtualenvs/wizard/lib/python3.7/site-packages/PyInstaller/loader/pyimod03_importers.py", line 627, in exec_module
    exec(bytecode, module.__dict__)
  File "site-packages/pkg_resources/__init__.py", line 86, in <module>
ModuleNotFoundError: No module named 'pkg_resources.py2_warn'
[14879] Failed to execute script runtime_raiden_contracts
[14879] LOADER: OK.
[14879] LOADER: Cleaning up Python interpreter.
[14878] LOADER: Restoring signal handlers
[14878] LOADER: freeing args
[14878] LOADER: returning child exit status 255
[14878] LOADER: Back to parent (RC: 255)
[14878] LOADER: Doing cleanup
[14878] LOADER: Freeing archive status for /home/oliver/Arbeit/github/raiden-wizard/raiden_wizard
```

## Solution
As discussed in [this](https://github.com/pypa/setuptools/issues/1963) issue the current version of setuptools does not fix this issue.

1) We could either implement a hotfix in the form of downgrading setuptools to an older version `pip install --upgrade 'setuptools<45.0.0'` (I tested this approach and it works).

2) Or we could add the missing package as hidden import 
```
a = Analysis(...,
             hiddenimports=['pkg_resources.py2_warn'],
             ...)
```
This can be achieved by adding `'pkg_resources.py2_warn'` to the list of [hidden imports in our raiden_webapp.spec file](https://github.com/raiden-network/raiden-wizard/blob/master/tools/pyinstaller/raiden_webapp.spec#L27).
(I also tested this approach and it works as well)

@jomuel since you're afaik the maintainer atm I'd leave the decision up to you. I'd prefer adding the missing package to the hidden imports.

**Comment by jomuel** [comment]
@eorituz I'm fine with either approach

**Comment by jomuel** [comment]
Should be fixed with #97
